coding/stylesheets.md

```diff
@@ -1,8 +1,12 @@
 ---
-title: BEM coding standard
+title: Stylesheets coding standard
 description: Code style standards for CSS and Sass
 ---
 
+## BEM methodology
+
+Our stylesheet conding standard is based on [BEM methodology](http://getbem.com/).
+
 For a quick recap on BEM, it works by breaking down all classes in a
```

Comment:
Again in the tone of taking ownership of our own guidelines, I think it might be nice to word this more like:

> All classes are broken down into one of three groups:

As I said, I think the "based on BEM methodology" in the line above should be quite sufficient.

Reply:
@nottrobin Thanks for comments. I updated the introduction text a bit.